packages/sitecore-jss-cli/src/process-env.ts

```diff
@@ -0,0 +1,26 @@
+import path from 'path';
+import * as dotenv from 'dotenv';
+import dotenvExpand from 'dotenv-expand';
+
+/**
+ * @param {string} dir The directory containing the .env files to load.
+ */
+export default function processEnv(dir: string) {
+  // replicate Next.js handling/behavior
+  // https://github.com/vercel/next.js/blob/v10.0.5/packages/next-env/index.ts#L80-L90
+  const mode = process.env.NODE_ENV;
+  const dotenvFiles = [
+    `.env.${mode}.local`,
```

Comment:
should this one also be excluded for test mode?

Reply:
Next.js doesn't, I took that code pretty much verbatim from them. Not sure the reasoning, but my main intention was to have similar behavior.